python/src/aiconfig/editor/client/src/reducers/aiconfigReducer.ts

```diff
@@ -150,6 +150,25 @@ export default function aiconfigReducer(
         prompts,
       };
     }
+
+    case "DELETE_OUTPUT": {
+      const prompts = state.prompts.map((prompt) => {
+        if (prompt._ui.id === action.id) {
+          return {
+            ...prompt,
+            outputs: undefined,
+          };
+        } else {
+          return prompt;
+        }
```

Comment:
We have a utils reducer method called `reduceReplacePrompt()`. Would prefer to use that to keep things cleaner and more abstracted

Reply:
+1, it will handle selecting the correct prompt by id